src/main/java/io/strimzi/kafka/quotas/StaticQuotaCallback.java

```diff
@@ -234,15 +250,15 @@ public static MetricName metricName(String name, String type, String group) {
      * @return the MetricName object derived from the arguments.
      */
     public static MetricName metricName(String name, String type, String group, LinkedHashMap<String, String> tags) {
-        final String tagValues = tags.entrySet().stream().map(entry -> String.format("%s=%s", sanitise(entry.getKey()), sanitise(entry.getValue()))).collect(Collectors.joining(","));
+        final String tagValues = tags.entrySet().stream().map(entry -> java.lang.String.format("%s=%s", sanitise(entry.getKey()), sanitise(entry.getValue()))).collect(Collectors.joining(","));
```

Comment:
Do we need to fullly qualify `String`?

Reply:
Resolved in https://github.com/strimzi/kafka-quotas-plugin/commit/82bf79507c7c1e76c40808e722e04847cb265ec6